README.md

```diff
@@ -1,2 +1,71 @@
-# Devpulse backend
+#### Devpulse backend
 ATLP Devpulse backend
+
+## Introduction
+This is backend system for the devpulse project. A project that is built to help players in the edu-tech sector manage students/trainees at ease. \
+This project is built using modern technologies that make it more scalable. Check the tech stack section. 
+
+## How to get started
+To start contributing to this project, follow the following steps to have the project up and running.
+Make sure you have the following boxes checked
+
+- [ ] A code editor installed (Visual Studio Code recommended and used mostly)
+- [ ] Git installed and configured on your system
+- [ ] MongoDB installed on your system (Optional)
+- [ ] Docker installed on your system (Highly recommended)
+
+If all the above are checked, use the following code snippet to get a copy of the repository
+
+```git
+git clone https://github.com/atlp-rwanda/atlp-pulse-bn.git
+cd atlp-pulse-bn
+npm install
+cp .env.example .env
+npm run dev
+
+// open the link printed at the end of the command
+```
+
+If you already have the project. Open a terminal from the project directory and be sure to update your local copy before proceeding using this command
+
+```git
+git pull --rebase --prune $@
+git submodule update --init --recursive
```

Comment:
Why are we using git submodule??

Reply:
Sorry, this ensure that when someone pulls change, all new branches that are not checkout locally are also added to the repo and they are updated accordingly. But there is no need for it in this regards since the user will have specific branch needed.

I will remove it